### PR TITLE
Feat slider

### PR DESCRIPTION
**Pull request by ankit9015**
Slider component in SocketUi has two variants: 
1 slider-primary
2. slider-secondary

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is really good. I liked the way you documented you html

File: components/simplified-grid/simplified-grid.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.grid-2,
+.grid-3 {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1rem;
+}
+
+.grid-item {
+  background-color: red;
+  min-height: 300px;
```
Comment (harshalini) [inline components/simplified-grid/simplified-grid.css:10]: You can use rem here instead of px

File: components/ratings/ratings.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.checked {
+  color: var(--gold-color);
+}
+
+.badge-ratings-container {
+  display: inline-flex;
+  background-color: var(--secondary-color);
+  gap: 5px;
+  padding: 0.5rem;
```
Comment (harshalini) [inline components/ratings/ratings.css:21]: If it is below 1 rem, you can use px

**Review by harshalini — APPROVED** [review]
Code is clean and understandable. I like the naming conventions.

File: components/simplified-grid/simplified-grid.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  background-color: blue;
+}
+
+@media (min-width: 760px) {
+  .grid-2 {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  .grid-3 {
+    grid-template-columns: repeat(3, 1fr);
+  }
```
Comment (t007rushi) [inline components/simplified-grid/simplified-grid.css:23]: is there a need for media queries in this part? can do without this for responsive

File: components/simplified-grid/simplified-grid.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    <script
+      src="https://kit.fontawesome.com/124ff44aed.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../components.css" />
+    <link rel="stylesheet" href="./simplified-grid.css" />
+  </head>
+  <body>
+    <div class="m-l p-s blue-container">
```
Comment (t007rushi) [inline components/simplified-grid/simplified-grid.html:17]: m-l p-s have clear naming for class names

File: components/slider/slider.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (t007rushi) [inline components/slider/slider.html:7]: title to update

File: components/toast/toast.css [hunk 6]
```diff
@@ -0,0 +1,40 @@
+html {
```
Comment (t007rushi) [inline components/toast/toast.css:1]: can put them in root for non-confliction as its good practices

File: components/toast/toast.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (t007rushi) [inline components/toast/toast.html:7]: same as above

File: components/navigation-dropdown/navigation-dropdown.css [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+  position: absolute;
+  margin-top: 5px;
+  min-width: 16rem;
+  padding: 10px 5px;
+  background-color: var(--white);
+  border: 1px solid var(--dark-gray);
+}
+
+.dropdown:hover .dropdown-button {
+  transform: scale(1.1);
```
Comment (t007rushi) [inline components/navigation-dropdown/navigation-dropdown.css:25]: liked the transformations you put in

**Review by t007rushi — APPROVED** [review]
Look for suggestions, else fine to merge

**Pull request merged**